specification/glossary.md

```diff
@@ -161,3 +164,25 @@ Pseudorandomly resolve flag values using a context property, such as a targeting
 ### Rule
 
 A rule is some criteria that's used to determine which variant a particular context should be mapped to.
+
+## SDK Paradigms
+
+Feature flag frameworks tend to come in two categories: those designed for use with a single user client application, and those designed for multi-user applications, such as web server applications. Some parts of the OpenFeature specification diverge depending on which paradigm the implementation seeks to adhere to.
+
+### Multi-Context Paradigm
```

Comment:
I'm not sure multi-context and single-context are the best names. I prefer dynamic-context and static-context instead. IMO that's the core distinction between the two paradigms - the context isn't fixed, and need to be provided each time.

Reply:
I think I agree. I will work on this wording.

Reply:
Addressed - dynamic/static